modules/pubsub/src/main/resources/reference.conf

```diff
@@ -1,7 +1,7 @@
 snowplow.defaults: {
   sources: {
     pubsub: {
-      parallelPullCount: 1
+      parallelPullFactor: 0.5
```

Comment:
If we want to match number of cores...shouldn't default be 1 then? Any specific reason it's 0.5?

Reply:
"Match" might have been the wrong word.  I will change the title to "Pubsub source scale the parallel pull count with number of cores".

On a 8 cpu instance, I found that 1 pull is not enough, 2 parallel pulls is _nearly_ enough, and with any larger number the metrics look the same.  So 4 seems a sensible number (factor = 0.5).